docs/tutorials/calibrating_armonk.ipynb

```diff
@@ -4,9 +4,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Calibrating single-qubit gates on Armonk\n",
+    "# Calibrating single-qubit gates on `ibmq_armonk`\n",
     "\n",
-    "In this tutorial we demonstrate how to calibrate single-qubit gates on Armonk using the Calibration framework in qiskit-experiments. We will run experiments to find the qubit frequency, calibrate the amplitude of DRAG pulses and chose the value of the DRAG parameter that minimizes leakage. The calibration framework requires the user to\n",
+    "In this tutorial we demonstrate how to calibrate single-qubit gates on `ibmq_armonk` using the Calibration framework in qiskit-experiments. We will run experiments to find the qubit frequency, calibrate the amplitude of DRAG pulses and chose the value of the DRAG parameter that minimizes leakage. The calibration framework requires the user to\n",
```

Comment:
Calibration should be lowercase?

Reply:
I wasn't sure either, but since you also had doubts, let's lower-case it.